tools/azure-sdk-tools/packaging_tools/auto_codegen.py

```diff
@@ -18,12 +18,13 @@ def main(generate_input, generate_output):
     spec_folder = data["specFolder"]
     sdk_folder = "."
     result = {}
+    python_tag = data.get('python_tag') if data.get('python_tag') else None
```

Comment:
```suggestion
    python_tag = data.get('python_tag')
```
None is default value:
![image](https://user-images.githubusercontent.com/70930885/173022470-6d0179e3-2baf-4144-b642-822f01664700.png)